qemu/tests/cfg/balloon_uniqueness.cfg

```diff
@@ -0,0 +1,17 @@
+- balloon_uniqueness:
+    type = balloon_uniqueness
+    balloon_dev_add_bus = 'yes'
+    variants:
+        - boot_with_two:
+            start_vm = no
+            balloon = balloon0 balloon1
+            balloon_dev_devid_balloon0 = balloon0
+            balloon_dev_devid_balloon1 = balloon1
+            boot_warning_msg = 'Only one balloon device is supported'
+        - plug_one:
+            balloon = balloon0
+            balloon_dev_devid_balloon0 = balloon0
+            hotplug = 1
```

Comment:
What does param 'hotplug' mean? Is it the times to do hotplug? Prefer a more suitable name.

Reply:
@huangyum What about `num_hotplug_devices`?

Reply:
It's better than 'hotplug'.